QA-404 Publish packages into separate directories ${DISTRO}-${RELEASE…

merlin-northern opened the pull request:
…}-${ARCH}.

ChangeLog: title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

merlin-northern commented:
@oleorhagen I just found out that `.template:publish:s3:apt-repo:` needs updating as well.

oleorhagen commented on .gitlab-ci.yml:
```diff
@@ -338,25 +338,25 @@ test:acceptance:
     - python3 -m pytest -v
       --mender-client-version $MENDER_VERSION
-      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/mender-client-deb-version)
+      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/debian-buster-amd64/mender-client-deb-version)
```
S:
Why not make a variable out of this while we're at it?

merlin-northern replied:
fixed

merlin-northern commented:
@oleorhagen: acceptance tests alignment is new: https://github.com/mendersoftware/mender-dist-packages/pull/228/files#diff-7fb11d01efb2e6788562533f26902ed0563b68bfbd51ccecf879ec3ad594f869 and I addressed your comments.
